src/Sylius/Behat/Context/Setup/Admin/ManagingCountriesContext.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Context\Setup\Admin;
+
+use Behat\Behat\Context\Context;
+use Sylius\Component\Addressing\Model\CountryInterface;
+use Sylius\Component\Core\Test\Services\SharedStorageInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+final class ManagingCountriesContext implements Context
+{
+    /**
+     * @var SharedStorageInterface
+     */
+    private $sharedStorage;
+
+    /**
+     * @var RepositoryInterface
+     */
+    private $countryRepository;
+
+    /**
+     * @param SharedStorageInterface $sharedStorage
+     * @param RepositoryInterface $countryRepository
+     */
+    public function __construct(SharedStorageInterface $sharedStorage, RepositoryInterface $countryRepository)
+    {
+        $this->sharedStorage = $sharedStorage;
+        $this->countryRepository = $countryRepository;
+    }
+
+    /**
+     * @Given /^the store has ("([^"]*)" country enabled)$/
+     */
+    public function theStoreHasCountryEnabled(CountryInterface $country)
+    {
+        $this->sharedStorage->set('country', $country);
+        $this->countryRepository->add($country);
+    }
+
+    /**
+     * @Given /^the store has ("([^"]*)" country disabled)$/
+     */
+    public function theStoreHasCountryDisabled(CountryInterface $country)
+    {
+        $this->sharedStorage->set('country', $country);
+        $this->countryRepository->add($country);
+    }
```

Comment:
Most likely it would work as one step with 2 tags, cause transformer is taking care of the difference here.